.github/workflows/ci.yml

```diff
@@ -76,8 +76,6 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v4
-        with:
```

Comment:
why remove?

Reply:
It is not clear why we need this.
LInter generates a warning: `Context access might be invalid: ref`
If you know why we need this - let's make a clarification comment.